.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       - name: Webpack build
         run: yarn run build
         env:
-          MITOPEN_API_BASE_URL: https://api.mitopen-test.odl.mit.edu
+          MITOL_API_BASE_URL: https://api.mitopen-test.odl.mit.edu
```

Comment:
Do we want to update these URLs while we're here or is that too many moving parts at the moment?

Reply:
It's a dummy value but I can update it.

Reply:
Well this one, yes. But in the production and rc files they are real. Thinking about it maybe they should come from ol-inf with much of the other config.

Reply:
That's going to depend on the timing, if I update those, the next build that goes out the frontend will try to connect to those immediately and things will break hard if they're not completely in place. I think maybe the answer is that they should come from a secret, not from a static value.

Reply:
Actually I'm inclined to say that changing the url values is out of scope for this issue. I'll leave the one I already touched but the others should get dealt with separately because otherwise it's comingling two different kinds of changes (variable name and value changes).

Reply:
Sounds good!